BUILD.bazel

```diff
@@ -347,15 +347,24 @@ test_suite(
 )
 
 test_suite(
-    name = "tests-vulkan-generator",
+    name = "lint",
     tests = [
-        # Linting
+        # __BEGIN_LINT
         "//vulkan_generator:lint",
+        "//vulkan_generator/codegen_utils:lint",
+        "//vulkan_generator/handle_remapper:lint",
         "//vulkan_generator/tests:lint",
         "//vulkan_generator/vulkan_parser:lint",
         "//vulkan_generator/vulkan_utils:lint",
+        # __END_LINT
+    ],
+)
 
-        # Tests
+test_suite(
+    name = "tests-vulkan-generator",
+    tests = [
+        # Run the linting as part of Vulkan generator tests
+        "//:lint",  # DO NOT REMOVE!
```

Comment:
Do we need a comment saying do not remove?

Reply:
Well, it is very easy to remove a test from bazel and not catch its missing. I just want to ensure that no one comes and says "why this is here" and remove it without thinking the implications. Having it there has no harm.